<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ili2dbOptions</class>
 <widget class="QDialog" name="Ili2dbOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ili2db Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="inheritance_group_box">
     <property name="title">
      <string>Inheritance type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="smart1_radio_button">
        <property name="toolTip">
         <string>Form the inheritance hierarchy with a dymamic strategy. The NewClass strategy is used for classes that are
referenced and whose base classes are not mapped using a NewClass strategy. Abstract classes are mapped
using a SubClass strategy. Concrete classes, without a base class or their direct base classes with a SubClass
strategy are mapped using a NewClass strategy. All other classes are mapped using a SuperClass strategy.</string>
        </property>
        <property name="text">
         <string>smart1Inheritance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="smart2_radio_button">
        <property name="toolTip">
         <string>Form the inheritance hierarchy with a dymamic strategy. Abstract classes are mapped using a SubClass
strategy. Concrete classes are mapped using a NewAndSubClass strategy.</string>
        </property>
        <property name="text">
         <string>smart2Inheritance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="toml_file_label">
        <property name="text">
         <string>Meta Attributes File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="toml_file_line_edit">
        <property name="toolTip">
         <string>Choose optional TOML file</string>
        </property>
        <property name="placeholderText">
         <string>[Optional]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toml_file_browse_button">
        <property name="toolTip">
         <string>Browse TOML files</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Ili2dbOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Ili2dbOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
